src/compiler/reviews.md

```diff
@@ -348,76 +270,63 @@ r? bootstrap
 ```
 
 ### bors
+We never merge PRs directly. Instead, we use bors. A qualified reviewer with bors privileges
+(e.g., a [compiler team member](./membership.md)) will leave a comment like `@bors r+`. This
+indicates that they approve the PR.
 
-We never merge PRs directly. Instead, we use bors. A qualified reviewer with
-bors privileges (e.g., a [compiler team member](./membership.md)) will leave a
-comment like `@bors r+`. This indicates that they approve the PR.
-
-People with bors privileges may also leave a `@bors r=username` command. This
-indicates that the PR was already approved by `@username`. This is commonly done
-after rebasing.
+People with bors privileges may also leave a `@bors r=username` command. This indicates that the
+PR was already approved by `@username`. This is commonly done after rebasing.
 
 Finally, in some cases, PRs can be "delegated" by writing `@bors delegate+` or
-`@bors delegate=username`. This will allow the PR author or the delegated user
-to approve the PR by issuing `@bors` commands like the ones above (but this
-privilege is limited to the single PR).
+`@bors delegate=username`. This will allow the PR author or the delegated user to approve the PR
+by issuing `@bors` commands like the ones above (but this privilege is limited to the single PR).
 
 ### Reverts
+If a merged PR is found to have caused a meaningful unanticipated regression, the best policy is
+to revert it quickly and re-land it later once a fix and regression test are added.
 
-If a merged PR is found to have caused a meaningful unanticipated regression,
-the best policy is to revert it quickly and re-land it later once a fix and
-regression test are added.
-
-A "meaningful regression" in this case is up to the judgment of the person
-approving the revert.
+A "meaningful regression" in this case is up to the judgment of the person approving the revert.
 
 Some criteria to consider if a revert is warranted:
```

Comment:
Discussion: hopefully unlikely in the future... but maybe accidental stabilizations?

Reply:
Fixed!